neo/conf/application.conf

```diff
@@ -93,7 +93,7 @@ asterixdb.url = "http://localhost:19002/query/service"
 asterixdb.lang = SQLPP
 
 # for the debug purpose, we can load a smaller json to speed up the front-end
-us.city.path = "/public/data/city.sample.json"
+us.city.path = "/public/data/city.json"
 
```

Comment:
@JavierJia changed to test something they developed(I guess for the count of tweets)